### PR TITLE
refactor(react-native): hangup and accept call button handlers from the SDK and DF app

### PR DESCRIPTION
**Pull request by khushal87**

### CODE REVIEW

File: sample-apps/react-native/dogfood/src/components/MeetingUI.tsx [hunk 1]
```diff
@@ -55,12 +50,9 @@ export const MeetingUI = ({ callId, navigation, route }: Props) => {
     }
   }, [call, appStoreSetState]);
 
-  const onHangUpCallButtonHandler = async () => {
+  const onCallHangupHandler = async () => {
     setShow('loading');
     try {
-      if (callingState === CallingState.LEFT) {
-        return;
-      }
```
Comment (vanGalilea) [inline sample-apps/react-native/dogfood/src/components/MeetingUI.tsx:63]: Is this not needed anymore, what is the story behind it?

**Comment by vanGalilea** [comment]
No regression ✅

File: sample-apps/react-native/dogfood/src/components/MeetingUI.tsx (hunk 1 above)
Comment (khushal87) [reply sample-apps/react-native/dogfood/src/components/MeetingUI.tsx:63]: Definitely needed. I removed it by mistake when doing the refactoring

File: packages/react-native-sdk/src/components/call/OutgoingCallView.tsx [hunk 2]
```diff
@@ -84,7 +103,7 @@ export const OutgoingCallView = ({
           </View>
 
           <CallControlsButton
-            onPress={cancelCallHandler.onPressHandler}
+            onPress={cancelCallHandler}
             color={theme.light.error}
             style={[styles.button, styles.cancelCallButton, theme.button.lg]}
             svgContainerStyle={[styles.svgContainerStyle, theme.icon.lg]}
```
[the remark this answers is not in the document]
Comment (khushal87) [reply packages/react-native-sdk/src/components/call/OutgoingCallView.tsx]: Yes, I have added the change.

**Review by vanGalilea — COMMENTED** [review]
APPROVED 🏁

**Pull request merged**